_posts/2024-02-27-replacing-default-admin-credentials.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title:  "Enhancing security in OpenSearch 2.12.0: The end of the default admin password"
+authors:
+ - dchanp
+date: 2024-02-27
+categories:
+ - community-updates
+ - technical-posts
+meta_keywords: "OpenSearch Security, Security Demo Configuration"
+meta_description: Learn about the changes to Security plugin's demo configuration setup
```

Comment:
Please update the meta with the following: 

**meta_keywords:**   OpenSearch security plugin, OpenSearch security,  OpenSearch 2.12.0, security configuration

**meta_description:**  Learn how OpenSearch 2.12.0 improves and secures the initial admin user setup processes using the demo configuration tool in the OpenSearch Security plugin.